sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
It would help with integration if we structure the projects like others in the repo.
Take a look at an existing project file ([Search](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/search/Azure.Search.Documents/src/Azure.Search.Documents.csproj) for example) to see the properties defined in the project file. Others can be dropped because they are defined in an imported .targets/.props file.